source/development/coredump.rst

```diff
@@ -0,0 +1,194 @@
+.. Copyright (C) 2024, Wazuh, Inc.
+
+.. meta::
+  :description: This section contains instructions to configure and collect coredumps for furture analisys
+
+.. _coredumps:
+
+HowTo configure Wazuh agent's OS to generate `coredumps`
+========================================================
+This section contains instructions to customize the diferent OS where Wazuh agent are installed.
```

Comment:
Singular - plural issues.
The plural of "OS" (operating system) can be either "OSes" or "OSs". Both are grammatically correct.
Subject/verb mismatch: "wazuh agents are installed" or "the wazuh agent is installed"

Reply:
It is not plural, it is possessive, it is referring to the SO, not the agent.

Reply:
Yes, I saw that and removed the comment, my mistake. The other one remains.